Formula/y/yt-dlp.rb

```diff
@@ -81,6 +81,9 @@ def install
   end
 
   test do
+    # Fails with bot detection
+    return if OS.linux? && ENV["HOMEBREW_GITHUB_ACTIONS"]
+
```

Comment:
@chenrui333 We could use our test fixture as a basic test.

```suggestion
    system bin/"yt-dlp", "https://raw.githubusercontent.com/Homebrew/brew/refs/heads/master/Library/Homebrew/test/support/fixtures/test.gif"

    # Fails with bot detection
    return if OS.linux? && ENV["HOMEBREW_GITHUB_ACTIONS"]